tutorials/eth-api/thirdweb.md

```diff
@@ -0,0 +1,285 @@
+
+![Set claim conditions](/images/tutorials/eth-api/thirdweb/thirdweb-3.webp "Setting claim conditions on thirdweb")
+
+
+### Mint Some NFTs {: #mint-some-nfts } 
+
+For aesthetic purposes, we'd like to have some NFTs show up in the marketplace that we created. Under the **Extensions** -> **NFTs** section, press **Claim**. Then, you can mint some NFTs by taking the following steps:
+
+1. Enter the address to receive the NFTs
+2. Enter the desired quantity to mint
+3. Press **Claim NFT** and confirm the transaction in your wallet
+
+![Mint some NFTs](/images/tutorials/eth-api/thirdweb/thirdweb-4.webp "Minting some NFTs on thirdweb")
+
+
+### Add NFT Contract to the dApp {: #add-nft-contract-to-the-dapp } 
+
+After deploying and minting your NFTs, you'll need to specify your NFT contract in `src/consts/nft_contract.ts`. First, add `moonbase` to the list of imports as follows: 
+
+```typescript title="nft_contracts.ts"
+import { moonbase, avalancheFuji, polygonAmoy, sepolia } from "./chains";
+```
+
+Then, add your NFT contract to the array of marketplace contracts as follows:
+
+```typescript title="nft_contracts.ts"
+  {
+    address: '0x5647fb3dB4e47f25659F74b4e96902812f5bE9Fb',
+    chain: moonbase,
+    title: 'Moonbase NFT',
+    thumbnailUrl:
+      'https://258c828e8cc853bf5e0efd001055fb39.ipfscdn.io/ipfs/QmTDyLBf2LaG6mzPniPjpX3P4DTFvjAk3gtUgrAb8EVPUF/2024-05-22%2008.17.59.jpg',
+    type: 'ERC721',
+  },
+```
+
+To get the IPFS URL of the image of your NFT that you uploaded when creating the NFT contract, head to the **Events** tab of your NFT contract and locate the `SharedMetadataUpdated` event. Expand the dropdown and you'll see the image URI. You can concatenate this to an IPFS CDN as shown above. 
+
+![Get IPFS URL](/images/tutorials/eth-api/thirdweb/thirdweb-5.webp "Getting the IPFS URL from thirdweb")
+
+The finished file can be viewed below:
+
+??? code "View nft-contracts.ts"
+    ```typescript
+    --8<-- 'code/tutorials/eth-api/thirdweb/nft-contracts.ts'
+    ```
+
+## Deploy Marketplace Contract {: #deploy-marketplace-contract }
+
+While the template includes existing marketplace contracts for a couple of testnets, let's deploy a similar one for Moonbase Alpha. Head to [thirdweb explore](https://thirdweb.com/explore){target=\_blank} and choose the `MarketplaceV3` contract. You can also access [`MarketplaceV3` directly](https://thirdweb.com/thirdweb.eth/MarketplaceV3){target=\_blank}. Press **Deploy Now**, then take the following steps:
+
+1. Add a name for the marketplace
+2. Select **Moonbase Alpha** as the network
+3. Press **Deploy Now**
+
+You'll be asked to confirm a transaction and provide a signature. The former deploys the marketplace contract and the latter adds the contract to your dashboard on thirdweb (which is not required but highly recommend for keeping track of your contracts).
+
+![Deploy marketplace contract](/images/tutorials/eth-api/thirdweb/thirdweb-6.webp "Deploying a marketplace contract on thirdweb")
```

Comment:
What is this synyax?

Reply:
With this formatting it creates hover text which is useful for people using accessibility software 

<img width="875" alt="Screenshot 2024-07-26 at 4 41 43 PM" src="https://github.com/user-attachments/assets/3d8588a2-181b-4472-84d2-ddc2c00f1b0b">